azavea/tilejson.io
# Add opacity and visibility controls to layers

## Overview

This PR adds visibility and opacity controls to the layers. It also adds visibility control to the base layer. Lastly, it adds these properties to the shared screen and shares the layers as is. 

Connects #42 

### Demo

<img width="1279" alt="screen shot 2017-11-17 at 10 21 55 pm" src="https://user-images.githubusercontent.com/3959096/32976532-c640908c-cbe5-11e7-9774-801432ae5a5b.png">

### Notes

The visibility and opacity of layers in diff mode is not covered here. I'm not sure if this is needed?

## Testing Instructions

 * Add layers and vary visibility and opacity
 * Share and check to see if the map was shared as is
